sdk/storage/azure-storage-common/inc/azure/storage/common/concurrent_transfer.hpp

```diff
@@ -36,7 +36,7 @@ namespace Azure { namespace Storage { namespace _internal {
           break;
         }
         int64_t chunkOffset = offset + chunkSize * chunkId;
-        int64_t chunkLength = (std::min)(length - chunkSize * chunkId, chunkSize);
+        int64_t chunkLength = std::min(length - chunkSize * chunkId, chunkSize);
```

Comment:
In the future, try to avoid making subtle changes like these in larger PRs, as they end up getting missed in code review. It's preferable to do them in separate smaller PRs :)